daml-script/export/integration-tests/golden/util.bzl

```diff
@@ -20,60 +20,71 @@ def daml_ledger_export_test(
         expected_daml,
         expected_args_json,
         expected_daml_yaml,
-        out_daml,
-        out_args_json,
-        out_daml_yaml,
-        dev = False):
-    # Disabled on Windows since postgres gets unhappy in client_server_build.
+        dev = False,
+        timeout = "short"):
+    # Disabled on Windows since postgres gets unhappy in client_server_test.
     if not is_windows:
-        actual_daml = name + "/" + out_daml
-        actual_args_json = name + "/" + out_args_json
-        actual_daml_yaml = name + "/" + out_daml_yaml
         server_dev_args = ["-C ledger.engine.allowed-language-versions=daml-lf-dev-mode-unsafe"] if dev else []
+        client_name = name + "-client"
 
-        client_server_build(
+        daml_ledger_export_test_client(
+            name = client_name,
+            expected_daml = expected_daml,
+            expected_args_json = expected_args_json,
+            expected_daml_yaml = expected_daml_yaml,
+        )
+
+        client_server_test(
             name = name,
-            outs = [
-                actual_daml,
-                actual_args_json,
-                actual_daml_yaml,
-            ],
-            client = "//daml-script/export/integration-tests/example-export-client",
+            client = client_name,
             client_args = [
                 "--target-port=%PORT%",
                 "--script-identifier={}".format(script_identifier),
                 "--party=" + ",".join(parties),
             ],
-            client_files = [dar],
+            client_files = ["$(rootpath {})".format(dar)],
```

Comment:
```suggestion
            client_files = ["$(rootpath %)" % dar],
```

Reply:
Personal preference, but I am fine you keep format if you really prefer it.

Reply:
ah that works better here, thanks